frontend/build/build-common.mjs

```diff
@@ -0,0 +1,13 @@
+export const buildOptions = {
```

Comment:
Nice work getting this all working - how did you find it in the end? It feels very clean to my eyes

Reply:
Thanks Jack! I'm glad it looks so clean, it took a lot of iterations and messing about haha. In the end, a lot was just getting rid of stuff I didn't need. The biggest pain point was what you see in the server.mjs from line 17 onward. Basically, using React Routes with esbuild doesn't play nice by default, so we have to add this slightly odd redirect logic as recommended [here](https://stackoverflow.com/questions/71132834/how-to-use-react-routes-with-esbuild). 

But it's definitely something I'm glad I spent time doing. It's given me a better idea of what's going on under the hood with these frameworks.

I found adding tailwind was also challenging, and I think I still have some issues with watching for changes there, but the more time I'm spending on these packaging/bundling issues the more comfortable I'm feeling.